mysite/code/controllers/ApiController.php

```diff
@@ -23,4 +23,23 @@ protected function formatResponse(array $data)
 
         return $response;
     }
+
+    /**
+     * Overrides this method only to prepend capturing any provided framework version header
+     *
+     * @inheritDoc
+     */
+    protected function handleAction($request, $action)
+    {
+        $frameworkVersionHeader = $request->getHeader('Silverstripe-Framework-Version');
```

Comment:
`SilverStripe`* 😄

Reply:
Can't capatalise anything except the beginning of a word in headers 😧 . I originally had `SS-Framework-Version` but it looked weird when it's sent as `Ss-...`

Reply:
Ok perfect, thanks